finalPrep/README.md

```diff
@@ -38,17 +38,17 @@ When you are finished, add screenshots of your calendar so we can provide feedba
 ### Mentorship Prep
 Mentorship is an integral part of the Turing experience and will help jumpstart your technical career. In order to get your mentor relationship started on the right foot, please complete the following deliverables:
 - [ ] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
-  - [ ] Add link to your gist here: 
+  - [ ] Add link to your gist here:https://gist.github.com/BrendaSotel0/3b12d5b81c7abc348616283e1c4f55fd
```

Comment:
Nice start here. I have a couple of tips for your letter to best represent yourself. Of course, you know yourself best, so no pressure!
- You can remove the kinds of things you might put in text messages, like any 'haha'. The informal tone risks making you come off as less serious. I think your personal nature will come across without those.
- I think getting worn thin by customer service is completely relatable, and you can avoid anything that you feel the need to follow up with 'That sounds bad!' I'd say focus on the fakeness of those positions wearing you thin, and the desire to collaborate with teammates in meaningful ways towards a shared goal that goes beyond just being a people-pleaser. 
- I wouldn't assume you will cause your mentor trouble- they signed up to help folks, and have been in your shoes! I understand feeling like a newbie to all of this, and feeling swamped with info- it is very likely your mentor will as well. I would steer a little more away from apology, and more towards 'Thank you in advance'. 

Again, just tips for consideration here! I really like that your voice and personality comes through in this.